CribblyBackend/appsettings.json

```diff
@@ -1,14 +1,25 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
+  "Serilog": {
+    "Using": [],
+    "MinimumLevel": {
+      "Default": "Debug",
+      "Override": {
+        "Microsoft": "Warning",
+        "System": "Warning"
+      }
+    },
+    "Enrich": ["FromLogContext", "WithMachineName", "WithProcessId", "WithThreadId"],
+    "WriteTo": [
+      { "Name": "Console" },
+      {
+        "Name": "RollingFile",
+        "Args": {
+          "pathFormat": "log\\BackendLog--{Date}.log",
```

Comment:
This also caused me a problem - unix uses `/` as a file separator, not `\`, so it would log to a file called `log\whatever.log`. Can you see if replacing `\\` with `/` is compatible with your system?

Reply:
`/` is working just fine!